### PR TITLE
fix(snuba): Fix the duration condition on the user_misery metric

### PR DESCRIPTION
**Pull request by fpacifici**
The user_misery aggregate is passing an infix expression to Snuba, which cannot be parsed.

### CODE REVIEW

File: src/sentry/api/event_search.py [hunk 1]
```diff
@@ -1174,7 +1174,7 @@ def has_default(self, params):
         "name": "user_misery",
         "args": [NumberRange("satisfaction", 0, None)],
         "calculated_args": [{"name": "tolerated", "fn": lambda args: args["satisfaction"] * 4.0}],
-        "transform": u"uniqIf(user, duration > {tolerated:g})",
+        "transform": u"uniqIf(user, greater(duration, {tolerated:g}))",
```
Comment (fpacifici) [inline src/sentry/api/event_search.py:1177]: @evanh I am not very familiar with this part of the code base. Is this goingn to impact the aggregation field for snuba only or others ?

**Comment by dashed** [comment]
@fpacifici Prior to this change, it seems user misery works fine on the product side. Does this PR fixes anything that was functionally broken?

**Comment by fpacifici** [comment]
It `seemd` to work properly. If you look at the Snuba dashboard there are plenty of warning of IncompleteParsing because of this. That's because the AST is not activated as mandatory so the query would return just fine, but fixing this is mandatory to be able to rollout the AST, at which point such queries would fail.

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Review by evanh — APPROVED** [review]

**Pull request merged**